jenkins-config/managed-config-files/custom/jjbini/customizedCredentialMappings.yaml

```diff
@@ -0,0 +1,6 @@
+---
+customizedCredentialMappings:
+  - credentialsId: "LF-JOBBUILDER"
+    tokenKey: "LF-JOBBUILDER"
```

Comment:
tokenKey should be "LFJOBBUILDER" since in the content file you are using "LFJOBBUILDER_USR"

Same for the sandbox token, it should be "LFJOBBUILDER-SBX"

Reply:
Done